spec/lib/cluster/discovery_spec.rb

```diff
@@ -7,5 +7,17 @@
     context 'has a discover method' do
       it { expect(discovery).to respond_to(:discover) }
     end
+
+    context 'can discover nodes', vcr: true do
+      it 'discovery with consul' do
+        consul = Cluster::Discovery.discover(
+          'consul',
+          consul_url: 'http://172.28.128.104:8500',
```

Comment:
Should this IP address by `ENV['TEST_CONSUL_HOST']` so it makes up with the Rakefile?